views/install.sh.erb

```diff
@@ -290,7 +292,7 @@ capture_tmp_stderr() {
 # do_wget URL FILENAME
 do_wget() {
   echo "trying wget..."
-  wget -O "$2" "$1" 2>$tmp_dir/stderr
+  wget -t 1 -T $download_timeout -O "$2" "$1" 2>$tmp_dir/stderr
```

Comment:
You probably want to only make --dns-timeout here more aggressive, not all of the timeouts?  also if you're truly offline won't this fail-fast anyway?